fixtures/cameo/ts60.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "ts60",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Anonymous"],
+    "createDate": "2021-05-27",
+    "lastModifyDate": "2021-05-27"
+  },
+  "links": {
+    "productPage": [
+      "https://www.cameolight.com/en/solutions/dj-musicians/static-lighting/theater-lights/11911/ts-60-w-rgbw",
+      "https://www.cameolight.com/en/solutions/dj-musicians/static-lighting/theater-lights/13839/ts-60-w-rgbw-wh"
+    ],
+    "manual": ["https://www.cameolight.com/en/downloads/file/id/1201324186"]
+  },
+  "physical": {
+    "dimensions": [145, 205, 240],
+    "weight": 3.6,
+    "power": 65,
+    "DMXconnector": "3-pin",
+    "lens": {
```

Comment:
The schema comment says 'PC', is this short for plano-convex? Which is preferred?

Reply:
Actually, I don't know; we inherited the usual values from QLC+. But this is a free-form property, so I'd say, let's keep it consistent with the manual.